docs/content/snippets/libpq-req.mdx

```diff
@@ -0,0 +1,5 @@
+:::info
+
+You need `libpq-dev` only if you plan to use the `--with-indexer` and `--with-graphql` flags with `sui start`. See [Local Network](/guides/developer/getting-started/local-network.mdx#start-the-local-network) for more information.
```

Comment:
Not sure if the flags are an && or || situation.

Reply:
--with-graphql requires --with-indexer, so I think it's fine.